src/content/blog/astro-tutorial.mdx

```diff
@@ -0,0 +1,48 @@
+---
+title: Astro Tutorial
```

Comment:
Would be cool to make this a bit more fun — “Astro Tutorial” is a bit bare. How do you feel about this?

```suggestion
title: Announcing the Astro Tutorial!
```

Reply:
I will commit, but took the liberty of editing out your `!` because no other titles have punctuation